Dockerfile.okd

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.16 AS builder
+FROM registry.ci.openshift.org/openshift/release:golang-1.17 AS builder
```

Comment:
you need to change line 6 below too

FROM registry.ci.openshift.org/origin/4.10:base

Reply:
if the e2e's stay green I'll handle this in a separate PR